spec/unit/module_spec.rb

```diff
@@ -28,7 +28,6 @@
 
   describe 'delegating to R10K::Module::Forge' do
     [ 'bar/quux',
-      'bar-quux',
```

Comment:
Why remove this?

Reply:
This was just testing the same thing twice.

Reply:
oh, I assumed it was testing that it can handle both name formats. but maybe something else is testing that more explicitly?

Reply:
hmm, my eyes must need to be checked because they just looked like both 'bar/quux` to me; i'll put it back.